contracts/feature-tests/composability/promises-features/src/call_sync_bt.rs

```diff
@@ -14,12 +14,58 @@ pub trait BackTransfersFeatureModule {
         token_nonce: u64,
         amount: BigUint,
     ) {
+        let ((), back_transfers) = self
+            .vault_proxy()
+            .contract(to.clone())
+            .retrieve_funds(token.clone(), token_nonce, amount.clone())
+            .execute_on_dest_context_with_back_transfers::<()>();
+
+        require!(
+            back_transfers.esdt_payments.len() == 1 || back_transfers.total_egld_amount != 0,
+            "Only one ESDT payment expected"
+        );
+
+        self.back_transfers_event(
+            &back_transfers.total_egld_amount,
+            &back_transfers.esdt_payments.into_multi_value(),
+        );
+    }
+
+    #[endpoint]
+    fn forward_sync_retrieve_funds_multi_call_bt(
```

Comment:
Please rename to `forward_sync_retrieve_funds_bt_twice`, "multi call" implies something else